### ISSUE TITLE
500 epoch replication for Resnet 50

### ISSUE THREAD

**Issue by gaopengcuhk** [facebookresearch/detr#115]
I replicate the Resnet 50 with 2 img per GPU, 8 GPU. However, my learning curve is quite different from yours.

<img width="229" alt="Screen Shot 2020-06-29 at 1 18 13 AM" src="https://user-images.githubusercontent.com/51525797/85953926-6f39f380-b9a6-11ea-89ec-5568974cb134.png">

**Comment by gaopengcuhk** [comment]
When I train models with 500 epochs, I observe overfitting.

**Comment by gaopengcuhk** [comment]
python -m torch.distributed.launch --nproc_per_node=8 --use_env main.py \
    --lr_drop 400 --epochs 500 \
    --coco_path /path/to/coco

**Comment by szagoruyko** [comment]
looks strange, I will try to replicate

**Comment by gaopengcuhk** [comment]
Any progress on this? @szagoruyko

**Comment by szagoruyko** [comment]
150 epochs done, still 8 days to go

**Comment by szagoruyko** [comment]
@gaopengcuhk below are training logs for R50 DETR on 4 nodes (blue) vs 1 node (orange), there is indeed overfitting, but the model reached 42/62 AP - close to 4 node training. Not sure what's going on, perhaps need to adjust hyperparameters like learning rate or grad clip value, will investigate if we have bandwidth
![image](https://user-images.githubusercontent.com/4953728/87524079-06ca6200-c688-11ea-97d9-9327a99610e8.png)

**Comment by linlililin123456** [comment]
@szagoruyko may i ask the batch size of 4 node training, is this log https://dl.fbaipublicfiles.com/detr/logs/detr-r50_log.txt tranined with 4 node of total batch size 64?
